CHANGELOG.md

```diff
@@ -16,7 +16,9 @@ Each change falls into one of the following categories: Added, Changed, Deprecat
 ### Added
  - Added `secp256k1_selftest`, to be used in conjunction with `secp256k1_context_static`.
 
-## [MAJOR.MINOR.PATCH] - YYYY-MM-DD
+## [0.1.0] - 2013-03-05 to 2021-12-25
+
+This version was never actually released.
```

Comment:
```suggestion
This version was in fact never released.
```
(actually sounds a little bit like it *was* released in some sense, just not properly...)

Reply:
fixed